### PR TITLE
chore: update README to fix contribution emojis

### PR DESCRIPTION
**Pull request by mihai-sysbio**
#### Main improvements in this PR:

This PR aims to fix the [broken emojis](https://github.com/SysBioChalmers/Human-GEM/blob/main/README.md#contributors) that appear in `README.md` in the `main` branch. [Here is how the _Contributors_ section should look like](https://github.com/SysBioChalmers/Human-GEM/blob/chore/redo-contributors/README.md#contributors).

**I hereby confirm that I have:**

- [ ] Tested my code on my own computer for running the model
- [ ] Selected `develop` as a target branch
- [ ] Any removed reactions and metabolites have been moved to the corresponding deprecated identifier lists

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,13 +13,13 @@ This repository contains the latest version of Human-GEM, a human genome-scale m
 
 If you use Human1 in your research, please cite:  
 
- > J. L. Robinson, P. Kocabas�, H. Wang, P.-E. Cholley, et al. An atlas of human metabolism. _Sci. Signal._ 13, eaaz1482 (2020). [doi:10.1126/scisignal.aaz1482](https://doi.org/10.1126/scisignal.aaz1482)
+ > J. L. Robinson, P. Kocabas�, H. Wang, P.-E. Cholley, et al. An atlas of human metabolism. _Sci. Signal._ 13, eaaz1482 (2020). [doi:10.1126/scisignal.aaz1482](https://doi.org/10.1126/scisignal.aaz1482)
```
Comment (mihai-sysbio) [inline README.md:16]: There seems to be a mishap in the formatting of Pinar's name, so my suggestion is to revert it:
```suggestion
 > J. L. Robinson, P. Kocabasÿ, H. Wang, P.-E. Cholley, et al. An atlas of human metabolism. _Sci. Signal._ 13, eaaz1482 (2020). [doi:10.1126/scisignal.aaz1482](https://doi.org/10.1126/scisignal.aaz1482)
```

File: README.md [hunk 2]
```diff
@@ -13,13 +13,13 @@ This repository contains the latest version of Human-GEM, a human genome-scale m
...
  
 Starting from Human-GEM v1.5.0, all the releases are also archived in [Zenodo](https://doi.org/10.5281/zenodo.4099692) from which specific version can be cited if used.
 
 If you use Mouse1, Rat1, Zebrafish1, Fruitfly1, or Worm1 in your research, please cite:   
 
-  > H. Wang, J. L. Robinson, P. Kocabas�, J. Gustafsson, M. Anton, P.-E. Cholley, et al. Genome-scale metabolic network reconstruction of model animals as a platform for translational research. _PNAS_ 118, e2102344118 (2021). [doi.org/10.1073/pnas.2102344118](https://doi.org/10.1073/pnas.2102344118)
+  > H. Wang, J. L. Robinson, P. Kocabas�, J. Gustafsson, M. Anton, P.-E. Cholley, et al. Genome-scale metabolic network reconstruction of model animals as a platform for translational research. _PNAS_ 118, e2102344118 (2021). [doi.org/10.1073/pnas.2102344118](https://doi.org/10.1073/pnas.2102344118)
```
Comment (mihai-sysbio) [inline README.md:22]:
```suggestion
  > H. Wang, J. L. Robinson, P. Kocabasÿ, J. Gustafsson, M. Anton, P.-E. Cholley, et al. Genome-scale metabolic network reconstruction of model animals as a platform for translational research. _PNAS_ 118, e2102344118 (2021). [doi.org/10.1073/pnas.2102344118](https://doi.org/10.1073/pnas.2102344118)
```

**Comment by mihai-sysbio** [comment]
Note: this PR should be squashed and merged directly to `main`.

File: README.md (hunk 1 above)
Comment (mihai-sysbio) [reply README.md]: I already committed the suggestion, but later thought maybe we should stick to the same formatting as in the published article, i.e. "Kocabas". What do others think?

**Review by haowang-bioinfo — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (mihai-sysbio) [reply README.md]: Alright, with @haowang-bioinfo's thumbs-up, the formatting has been adjusted in [8cffe89](https://github.com/SysBioChalmers/Human-GEM/pull/647/commits/8cffe891da07cae9cccf0b985eb1283da8537c79).

**Pull request merged**